hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -366,8 +367,10 @@ private Stream<HoodieBaseFile> convertFileStatusesToBaseFiles(FileStatus[] statu
    * @param statuses List of FIle-Status
    */
   private Stream<HoodieLogFile> convertFileStatusesToLogFiles(FileStatus[] statuses) {
-    Predicate<FileStatus> rtFilePredicate = fileStatus -> fileStatus.getPath().getName()
-        .contains(metaClient.getTableConfig().getLogFileFormat().getFileExtension());
+    Predicate<FileStatus> rtFilePredicate = fileStatus ->  {
+      Matcher matcher = FSUtils.LOG_FILE_PATTERN.matcher(fileStatus.getPath().getName());
+      return matcher.find();
```

Comment:
Should we check for both this pattern as well as log file extension?

Reply:
makes sense. will fix it.